boards/FSM/Src/App/states/App_DriveState.c

```diff
@@ -0,0 +1,55 @@
+//self definition
+#include "states/App_DriveState.h"
+
+//to go to
+#include "states/App_FaultState.h"
+
+//function to run.
+#include "states/App_AllStates.h"
+#include "App_SharedMacros.h"
+
+
+void App_DriveStateRunOnEntry(struct StateMachine *const state_machine){
+    return;
+}
+
+void App_DriveStateRunOnTick1Hz(struct StateMachine *state_machine){
+    App_AllStatesRunOnTick1Hz(state_machine);
+}
+void App_DriveStateRunOnTick100Hz(struct StateMachine *state_machine){
+    App_AllStatesRunOnTick100Hz(state_machine);
+
+    struct FsmWorld *         world              = App_SharedStateMachine_GetWorld(state_machine);
+    struct FsmCanTxInterface *can_tx             = App_FsmWorld_GetCanTx(world);
+    struct AcceleratorPedals *accelerator_pedals = App_FsmWorld_GetPappsAndSapps(world);
+    struct Brake *            brake              = App_FsmWorld_GetBrake(world);
+    struct Coolant *          coolant            = App_FsmWorld_GetCoolant(world);
+    struct Steering *         steering           = App_FsmWorld_GetSteering(world);
+    struct Wheels *           wheels             = App_FsmWorld_GetWheels(world);
+
+    // NEW ALL STATES CODE
+    bool coolantTriggerShutdown = false;
+    App_AcceleratorPedals_Broadcast(can_tx, accelerator_pedals, brake);
```

Comment:
Shouldn't this be in all-states? We want to be broadcasting regardless of state

Reply:
Fixed in c210ce64d3385ef8fd1aa4f1e796a9d9e922483a